<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditServer</class>
 <widget class="QFrame" name="EditServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>127</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="lbCaption">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AnimatedLineEdit" name="edTittle">
     <property name="placeholderText">
      <string>Tittle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wtAddressSpacing" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AnimatedLineEdit" name="edAddress">
     <property name="placeholderText">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wtPortSpacing" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AnimatedLineEdit" name="edPort">
     <property name="styleSheet">
      <string notr="true">#edPort {} </string>
     </property>
     <property name="placeholderText">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wtButtons" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="btCancel">
        <property name="text">
         <string>CANCEL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wtButtonsSpacing" native="true"/>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="btSave">
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">controls/AnimatedLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
